src/pages/Home/VideoItem.component.spec.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import VideoItem from './VideoItem.component';
+import videos from '../../mock/youtube-videos-mock.json';
+
+// test('render VideoItem', () => {
+
+//     const component = render(<VideoItem item={item} />)
+
+//     component.getByRole('img');
+
+// })
+
+jest.mock('react-router-dom', () => {
+  return {
+    Link: jest.fn(({ children }) => <div role="link">{children}</div>),
+  };
+});
```

Comment:
this is a clever way to deal with browser router, did you tried wrapping the component on the
```js
beforeEach(() => {
    render(
     <BrowserRouter>
       <VideoItem item={item} />
     </BrowserRouter>
    );
  });
```